Got this error:

https://github.com/mlflow/mlflow/pull/3855/checks?check_run_id=1563229497#step:4:162

```
E           AttributeError: <module 'mlflow.pytorch' from '/home/runner/work/mlflow/mlflow/mlflow/pytorch/__init__.py'> does not have the attribute 'safe_patch'
```


In pytorch autlogging, `safe_patch` is imported & called in `mlflow/pytorch/_pytorch_autolog.py`.

--- a/mlflow/pytorch/_pytorch_autolog.py
+++ b/mlflow/pytorch/_pytorch_autolog.py
@@ -8,9 +8,7 @@ from pytorch_lightning.core.memory import ModelSummary
 from pytorch_lightning.utilities import rank_zero_only
 
 from mlflow.pytorch import FLAVOR_NAME
-from mlflow.utils.annotations import experimental
 from mlflow.utils.autologging_utils import (
-    autologging_integration,
     safe_patch,
     ExceptionSafeAbstractClass,
     try_mlflow_log,

--- a/mlflow/utils/autologging_utils.py
+++ b/mlflow/utils/autologging_utils.py
@@ -409,7 +409,7 @@ def exception_safe_function(function):
 
 def _exception_safe_class_factory(base_class):
     """
-    Creates an exception safe class that inherits from `base_class`.
+    Creates an exception safe metaclass that inherits from `base_class`.
     """
 
     class _ExceptionSafeClass(base_class):

--- a/pylintrc
+++ b/pylintrc
@@ -147,6 +147,7 @@ disable=print-statement,
         # redefined-outer-name disabled to avoid linting errors when using pytest fixtures
         # (see https://stackoverflow.com/questions/46089480/pytest-fixtures-redefining-name-from-outer-scope-pylint)
         redefined-outer-name,
+        invalid-metaclass,
 
 # Enable the message, report, category or checker with the given id(s). You can
 # either give multiple identifier separated by comma (,) or put this option

